pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -510,4 +510,20 @@ self: super: builtins.intersectAttrs super {
   LDAP = dontCheck (overrideCabal super.LDAP (drv: {
     librarySystemDepends = drv.librarySystemDepends or [] ++ [ pkgs.cyrus_sasl.dev ];
   }));
+
+  futhark = with pkgs;
+    let path = stdenv.lib.makeBinPath [ gcc ];
+    in overrideCabal (addBuildTool super.futhark makeWrapper) (_drv: {
+      postInstall = ''
+        wrapProgram $out/bin/futhark-c \
+          --prefix PATH : "${path}"
+
+        wrapProgram $out/bin/futhark-opencl \
```

Comment:
@basvandijk These wrapper don't seem to be applied like this, building either `pkgs.futhark` or `pkgs.haskellPackages.futhark` results in raw ELF binaries

Reply:
Strange, I did test if it created the correct wrappers but I can confirm it now results in ELF binaries. I'll look into it.

Reply:
This should now be fixed by https://github.com/NixOS/nixpkgs/pull/43865/commits/2cdeee2367ccc478b25a6e1db52d099e3d258896 which I pushed to this PR.

@peti OK with this solution?